examples/reservoir_simulation/Norne/README.md

```diff
@@ -564,8 +564,8 @@ cd ..
 ##### Bare Metal
 ```bash
 conda activate MDLO
-wget --content-disposition https://api.ngc.nvidia.com/v2/resources/nvidia/modulus/modulus_reservoir_simulation_supplemental_material/versions/latest/zip -O modulus_reservoir_simulation_supplemental_material_latest.zip
-unzip modulus_reservoir_simulation_supplemental_material_latest.zip
+wget --content-disposition https://api.ngc.nvidia.com/v2/resources/nvidia/modulus/modulus_reservoir_simulation_supplemental_material/versions/0.1/zip -O modulus_reservoir_simulation_supplemental_material_0.1.zip
```

Comment:
we call versions of models `v0.1` and the versions of resources here `0.1`. Not that big of a deal just inconsistent a little.

Reply:
Good point! I think as we publish more resources, we need to standardize. I will keep this one as is but will make a note for future updates!